tensilelite/Tensile/Source/lib/source/Debug.cpp

```diff
@@ -188,6 +188,10 @@ namespace Tensile
         const char* exp_select = std::getenv("TENSILE_EXPERIMENTAL_SELECTION");
         if(exp_select)
             m_experimentSelection = strtol(exp_select, nullptr, 0) != 0;
+        
+        const char* sk_select = std::getenv("TENSILE_STREAMK_LIBRARY");
```

Comment:
Both TENSILE_EXPERIMENTAL_SELECTION  and TENSILE_STREAMK_LIBRARY try to set vale in m_experimentSelection.
Could you integrate TENSILE_STREAMK_LIBRARY into TENSILE_EXPERIMENTAL_SELECTION directly?

Reply:
We've been asked explicitly to avoid using EXPERIMENTAL variable since QA will not run any experimental labelled features

Reply:
If so, I would recommend that you just rename "TENSILE_EXPERIMENTAL_SELECTION" as other.
ex: TENSILE_SOLUTION_SELECTION_METHOD or TENSILE_SOLUTION_SELECTION_ALGORITHM...

Reply:
env variable renamed as requested